src/browser.ts

```diff
@@ -235,56 +216,74 @@ async function registerPlugins(
   return ctx
 }
 
-export class AnalyticsBrowser {
-  static async load(
-    settings: AnalyticsBrowserSettings,
-    options: InitOptions = {}
-  ): Promise<[Analytics, Context]> {
-    // this is an ugly side-effect, but it's for the benefits of the plugins that get their cdn via getCDN()
-    if (settings.cdnURL) setGlobalCDNUrl(settings.cdnURL)
-    const legacySettings =
-      settings.cdnSettings ??
-      (await loadLegacySettings(settings.writeKey, settings.cdnURL))
-
-    const retryQueue: boolean =
-      legacySettings.integrations['Segment.io']?.retryQueue ?? true
-
-    const opts: InitOptions = { retryQueue, ...options }
-    const analytics = new Analytics(settings, opts)
-
-    const plugins = settings.plugins ?? []
-    Context.initMetrics(legacySettings.metrics)
-
-    // needs to be flushed before plugins are registered
-    flushPreBuffer(analytics)
-
-    const ctx = await registerPlugins(
-      legacySettings,
-      analytics,
-      opts,
-      options,
-      plugins
-    )
+async function loadAnalytics(
```

Comment:
Moved this outside the class to make it "private private". When we enable ES2015 in tsconfig, we can use _real_ private fields.